cypress/e2e/smoke/tx_history.cy.js

```diff
@@ -131,4 +131,25 @@ describe('Transaction history', () => {
         cy.contains('span', 'Success').should('exist')
       })
   })
+
+  it('should expand/collapse all actions', () => {
+    // Open the tx details
+    cy.contains('div', 'Mar 23, 2023')
+      .next()
+      .click()
```

Comment:
Can you verify that before the click the elements are not visible? (Same as you do after clicking "Collapse all")